test/arrays/methods/internal/resizeAllocRange.chpl

```diff
@@ -0,0 +1,33 @@
+/* Test to exercise the Grow code paths in _array.resizeAllocRange */
```

Comment:
As a hedge against arrayAsVecGrowthFactor changing in the future and breaking this test you could explicitly set it to 1.5 in a .compopts file.

Reply:
good call

Reply:
This looks good to me assuming your testing is clean.

Reply:
Cool - paratesting now!